test/core/simd/meta/simd_f64x2_arith.py

```diff
@@ -101,110 +101,59 @@ def combine_binary_arith_test_data(self):
             ]
         }
 
-    def get_normal_case(self):
-        return super().get_normal_case().replace('nan_f32x4', 'nan_f64x2')
-
     def get_invalid_cases(self):
         return super().get_invalid_cases().replace('32', '64')
 
     @property
     def mixed_sqrt_nan_test_data(self):
```

Comment:
why does this function have `sqrt` in the name but many different operations in its data?

Reply:
Yeah, this is for combination testing, the name really does not make sense, I will update it. :)

Reply:
Done.